test-harness/specs/security-scheme-validation-AC-5_failing.oas2.txt

```diff
@@ -0,0 +1,30 @@
+====test====
+Making a request (and including invalid credentials) to an endpoint,
```

Comment:
Assumed: since 403 examples ended up in the reviewed ACs, I came up with this. But not sure if should do so (?).
How would we know that dGVzdDp0ZXN0 is valid?

Reply:
Invalid can mean a lot of things I guess, this might have been lost in translation from notes to examples, and now be an impossible test. We want to make sure there’s tests for missing, or wrong type of token, or if it wants bearer and you don’t specify the bearer keyword, etc, but no the actual token is opaque and we don’t know what is valid or not.